components/tools/OmeroWeb/omeroweb/webclient/views.py

```diff
@@ -372,9 +372,8 @@ def logout(request, conn=None, **kwargs):
 
 
 ###########################################################################
-@login_required()
-@render_response()
-def load_template(request, menu, conn=None, url=None, **kwargs):
+def _load_template(request, menu, conn=None, url=None, **kwargs):
```

Comment:
@will-moore I made a clean up in load_template and now allow to override it. Could you please have a look if that won't break anything?